audit.tf

```diff
@@ -122,6 +123,12 @@ resource "aws_guardduty_detector" "audit" {
   }
 }
 
+resource "aws_guardduty_organization_configuration" "audit" {
+  auto_enable = true
+  detector_id = aws_guardduty_detector.audit[0].id
+}
+
+
```

Comment:
```suggestion

```

Can you pls remove the double line here.

Reply:
Fixed!